net/host.go

```diff
@@ -19,9 +19,11 @@ import (
 )
 
 const (
-	// ProtocolID is the base atomic swap network protocol ID.
-	ProtocolID     = "/atomic-swap/0.2"
-	maxMessageSize = 1 << 17
+	// ProtocolID is the base atomic swap network protocol ID prefix. The full ID
+	// includes the chain ID at the end.
+	ProtocolID          = "/atomic-swap/0.2"
+	maxMessageSize      = 1 << 17            // TODO: implement separate sizes for query/relay protocols
+	maxRelayMessageSize = maxMessageSize / 2 // TODO: optimize
```

Comment:
Reminder to self: Look at relayer message sizes to figure out what a good value above would be.